modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/jdbc/JdbcConnectionPropertiesTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.runner.app.jdbc;
+
+import java.sql.DriverPropertyInfo;
+import org.apache.ignite.internal.jdbc.ConnectionPropertiesImpl;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * {@link ConnectionPropertiesImpl} unit tests.
+ */
+public class JdbcConnectionPropertiesTest {
+    /**
+     * Test check the {@link ConnectionPropertiesImpl#getDriverPropertyInfo()} return properties with prefix {@link
+     * ConnectionPropertiesImpl#PROP_PREFIX}
+     */
+    @Test
+    public void testNamePrefixDriverPropertyInfo() {
+        ConnectionPropertiesImpl connProps = new ConnectionPropertiesImpl();
+        DriverPropertyInfo[] propsInfo = connProps.getDriverPropertyInfo();
+
+        String propName = propsInfo[0].name;
```

Comment:
Why only the first property is checked?

Reply:
Refactored test. Now it checks every property.